.github/workflows/ci.yml

```
@@ -28,3 +28,6 @@ jobs:
         uses: microsoft/action-python@0.7.0
         with:
           pylint: true
+
+      - name: Pytest
+        run: python -m pytest
```

```suggestion
        uses: microsoft/action-python@0.7.0
        with:
          testing: true
```

:pick: The microsoft action includes pytest as one of the actions it supports, I'm not sure if it's really any different but it might include better integration with Github that I don't want to figure out myself :smile:

---

Absolutely! I'll try it out, thanks!